Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/JobDataInteraction/EntityPersistentDataWriter.cs

```diff
@@ -30,5 +25,20 @@ internal EntityPersistentDataWriter(ref UnsafeParallelHashMap<Entity, TData> loo
             get => m_Lookup[entity];
             set => m_Lookup[entity] = value;
         }
+
+
+        internal EntityPersistentDataWriter(ref UnsafeParallelHashMap<Entity, TData> lookup)
+        {
+            m_Lookup = lookup;
+        }
+
+        /// <inheritdoc cref="UnsafeParallelHashMap{TKey,TValue}.Remove"/>
+        public bool Remove(Entity entity) => m_Lookup.Remove(entity);
+
+        /// <inheritdoc cref="UnsafeParallelHashMap{TKey,TValue}.TryGetValue"/>
+        public bool TryGet(Entity entity, out TData data) => m_Lookup.TryGetValue(entity, out data);
```

Comment:
```suggestion
        public bool TryGetValue(Entity entity, out TData data) => m_Lookup.TryGetValue(entity, out data);
```
Minor - consistency in function name

Reply:
hmm, I suppose both of these suggestions come down to. Does the writer HAVE a lookup or is it a lookup?
Key and value aren't really concepts from the consuming side of this type.

It's really used more like a CDFE in usage.

I'm not against the change just wanted to throw out the rationale before making the change.

Reply:
We talked about it, it's ok all good